hack/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -eux
+
+# Required operator-sdk version
+REQUIRED_OPERATOR_SDK_VERSION="v0.12.0"
+
+if [ ! "$(command -v operator-sdk)" ]; then 
+    LOCAL_OPERATOR_SDK_VERSION=""
+else 
+    LOCAL_OPERATOR_SDK_VERSION="$(operator-sdk version | head -n1 | cut -d" " -f3 | tr -d '",')"
+fi
+
+# If local operator-sdk version is not required version
+# or operator-sdk is not installed, install operator-sdk v0.12.0
+if [ "${LOCAL_OPERATOR_SDK_VERSION}" != "${REQUIRED_OPERATOR_SDK_VERSION}" ] || [ -z "${LOCAL_OPERATOR_SDK_VERSION}" ]; then
+    curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${REQUIRED_OPERATOR_SDK_VERSION}/operator-sdk-${REQUIRED_OPERATOR_SDK_VERSION}-x86_64-linux-gnu
+    chmod +x operator-sdk-${REQUIRED_OPERATOR_SDK_VERSION}-x86_64-linux-gnu
+    sudo mkdir -p /usr/local/bin/
+    sudo cp operator-sdk-${REQUIRED_OPERATOR_SDK_VERSION}-x86_64-linux-gnu /usr/local/bin/operator-sdk
```

Comment:
Is it necessary to use sudo here? You could just install in `${HOME}/bin/`. Personally I installed the sdk using `go get` so it's in `~/go/bin/`; that would be another way to do it (and is not specific to a particular operating system and CPU arch like downloading the tarball is, so it would work for e.g. Mac users as well).

Reply:
Yes, please use "go get" like we do for kustomize over in the cluster-api-provider repo https://github.com/metal3-io/cluster-api-provider-baremetal/blob/master/Makefile#L19

Reply:
I modified install.sh to install operator-sdk to ./bin folder